### PR TITLE
Add icon as button dark option

### PR DESCRIPTION
**Pull request by aju**
closes https://github.com/brainly/style-guide/issues/470
screen shot 2016-01-05 at 16 13 37

### CODE REVIEW

**Comment by vergilius** [comment]
:ok:

**Comment by vergilius** [comment]
:+1:

File: src/components/buttons/icon-as-button.html [hunk 1]
```diff
@@ -17,6 +17,13 @@ <h3 class="docs-block__header">Default</h3>
             </svg>
           </div>
         </button>
+      <button class="mint-icon-as-button mint-icon-as-button--dark">
+        <div class="mint-icon-as-button__hole">
+          <svg class="mint-icon mint-icon--adaptive mint-icon--x26">
+            <use xlink:href="#icon-heart"></use>
+          </svg>
+        </div>
+      </button>
```
Comment (kdzwinel) [inline src/components/buttons/icon-as-button.html]: wrong indentation - install editorconfig :/

Comment (aju) [inline src/components/buttons/icon-as-button.html]: HTML has 2 spaces and here you have 2 spaces :P

**Comment by aju** [comment]
Talked with @kdzwinel and we will make separate PR with fixing spacings

**Pull request merged**